src/geo_types.rs

```diff
@@ -0,0 +1,53 @@
+///```rust
+/// # use assert_approx_eq::assert_approx_eq;
+/// extern crate proj;
+/// use proj::Proj;
+/// use geo_types::Coordinate;
+///
+/// let from = "EPSG:2230";
+/// let to = "EPSG:26946";
+/// let nad_ft_to_m = Proj::new_known_crs(&from, &to, None).unwrap();
+/// let result = nad_ft_to_m
+///     .convert(Coordinate { x: 4760096.421921f64, y: 3744293.729449f64 })
+///     .unwrap();
+/// assert_approx_eq!(result.x, 1450880.29f64, 1.0e-2);
+/// assert_approx_eq!(result.y, 1141263.01f64, 1.0e-2);
+/// ```
+impl<T: crate::proj::CoordinateType> crate::Point<T> for geo_types::Coordinate<T> {
```

Comment:
Note - this PR would also let you do Proj stuff with Coords, not just points.

Reply:
Now that I'm typing it out - we could take this even further and have a `Projectable` trait or somesuch that just required implementing a Iter<Item=proj::Point<T>>, then we could get easy projection of all the geo-type's. 🤔

Reply:
I'm not going to pursue the Iter scenario in this PR, but would like to do it in some followup work.